docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -9645,4 +9645,4 @@ and these instructions on [developer setup and contributing code](https://github
 You can report issues with nightly preview builds in the following ways:
 - Report issues in our [github issues list](https://github.com/azure/azure-cli/issues/)
 - Contact the product team at [azfeedback@microsoft.com](mailto:azfeedback@microsoft.com)
-- Provide feedback from the command line with the `az feedback` command
\ No newline at end of file
+- Provide feedback from the command line with the `az feedback` command
```

Comment:
Seems you add a new blank line after this last line, please check and remove it.